runtests.py

```diff
@@ -2447,8 +2449,8 @@ def main():
         "--no-capture", dest="capture", default=True, action="store_false",
         help="do not capture stdout, stderr in srctree tests. Makes pdb.set_trace interactive")
     parser.add_argument(
-        "--limited-api", dest="limited_api", default=False, action="store_true",
-        help="Compiles Cython using CPython's LIMITED_API")
+        "--limited-api", dest="limited_api", nargs='?', default=False, const=True, action="store",
+        help="Compiles Cython using CPython's LIMITED_API. You can optional specify a version in the form '3.11'")
```

Comment:
```suggestion
        help=("Compiles Cython using CPython's LIMITED_API. "
              "Accepts an optional version in the form '3.11', otherwise uses current."))
```